tasks/us/census/tiger.py

```diff
@@ -664,10 +666,10 @@ def populate(self):
         session = current_session()
         stmt = ('INSERT INTO {output} '
                 'SELECT geoid, ST_Union(ST_MakePolygon(ST_ExteriorRing(the_geom))) AS the_geom, '
-                '       MAX(aland) AS aland, cdb_observatory.FIRST(name) AS name '
+                '       MAX(aland) AS aland '
```

Comment:
I remember speaking about a _missing_ `name` column. Could you give me some context about why is this one not needed, please?